app/views/content/a-day-in-the-life-of-a-teacher.md

```diff
@@ -94,7 +94,8 @@ $q-sarah-geography-2$
 
 <p>To get one step closer to teaching you can:</p> 
   <ul>
-    <li>get an <a href="/explore-teaching-advisers">explore teaching adviser</a> who can help you discover if teaching is for you and find classroom experience</li>
+    <li>get an <a href="/explore-teaching-advisers">explore teaching adviser</a> who can answer all your questions about teaching and help you find classroom experience</li>
+    <li>explore <a href="/is-teaching-right-for-me">if teaching is right for you</a></li>
```

Comment:
Is there another word we can use other than `explore` in this bullet @EllieNodder? Slightly confusing when we have just talked about explore teaching advisers in the previous bullet?